src/ui/mainwindow.cpp

```diff
@@ -1163,17 +1163,29 @@ void MainWindow::MediaPlaying() {
       IconLoader::Load("media-playback-pause", IconLoader::Base));
   ui_->action_play_pause->setText(tr("Pause"));
 
-  bool enable_play_pause = !(app_->player()->GetCurrentItem()->options() &
-                             PlaylistItem::PauseDisabled);
+  const PlaylistItemPtr item = app_->player()->GetCurrentItem();
+
+  bool enable_play_pause = !(item->options() & PlaylistItem::PauseDisabled);
   ui_->action_play_pause->setEnabled(enable_play_pause);
 
-  bool can_seek = !(app_->player()->GetCurrentItem()->options() &
-                    PlaylistItem::SeekDisabled);
+  bool can_seek = !(item->options() & PlaylistItem::SeekDisabled);
   ui_->track_slider->SetCanSeek(can_seek);
 
   // We now always enable Love when playing since it works for local files
   ui_->action_love->setEnabled(true);
 
+  // Set the rate/love icon
+  if (item) {
```

Comment:
We've already dereferenced `item` by this point. Maybe better to check for nullptr, log an error, and return at line 1167?

Reply:
Good point, will do